build.csx

```diff
@@ -266,3 +269,32 @@ public string GetVersion()
     var buildSuffix = versionSuffix + "-build" + build.PadLeft(6, '0');
     return version + buildSuffix;
 }
+
+public string GetVSLocation()
+{
+    using (var process = new Process())
+    {
+        process.StartInfo = new ProcessStartInfo
+        {
+            FileName = $"\"{vswhere}\"",
+            Arguments = "-nologo -latest -property installationPath -requires Microsoft.Component.MSBuild -version [15,16)",
+            WorkingDirectory = ".",
+            UseShellExecute = false,
+            RedirectStandardOutput = true
+        };
+
+        process.Start();
+        process.WaitForExit();
+        if (process.ExitCode != 0)
+        {
+            throw new InvalidOperationException($"The vswhere command exited with code {process.ExitCode}.");
+        }
+        else
+        {
+            string installationPath = process.StandardOutput.ReadToEnd().Trim();
+            if (string.IsNullOrEmpty(installationPath))
+                throw new InvalidOperationException("Visual Studio 2017 was not found");

```

Comment:
I'm a stickler, but we have moustaches up on lines 289 and 291. Could we here as well?